### DISCUSSION TITLE
Blending prompts e.g. [dog|elephant]

### DISCUSSION THREAD

**Discussion by bblackmoor** [lllyasviel/Fooocus#2322] [Q&A] [answered]
I thought that I could blend prompts by placing them between square brackets and separating the prompts with pipes, like so:
[Elizabeth Olsen|Anya Taylor-Joy|Charlize Theron]

The output would look like a blend of Elizabeth Olsen, Anya Taylor-Joy, and Charlize Theron. Am I using the wrong syntax?

Thank you.

**Comment by mashb1t** [comment, marked as answer]
@bblackmoor this syntax is not (yet) supported in Fooocus, sorry to say.

**Comment by bblackmoor** [comment]
@mashb1t ... That is unfortunate. What do _you_ think is the easiest GUI for SDXL which **has** this feature? Anyone have a suggestion?

**Reply by mashb1t** [reply]
@bblackmoor A1111 has prompt attention shift, i'm not certain if mixing is possible in this exact syntax.

**Reply by bblackmoor** [reply]
I will use whatever syntax is required. :)

**Reply by mashb1t** [reply]
what you can do in Fooocus is use multiline prompts, each line with a separate concept/person.

In A1111 there is the AND keyword, see https://www.reddit.com/r/StableDiffusion/comments/xwplfv/and_prompt_combinations_just_landed_in/, as well as attention shift like this https://www.nextdiffusion.ai/tutorials/prompt-editing-in-stable-diffusion-merging-images-seamlessly

**Reply by bblackmoor** [reply]
"what you can do in Fooocus is use multiline prompts, each line with a separate concept/person."

My understanding was that multiline prompts alternate the generated images, rather than alternating _during_ each generation. Is that not so?

My goal is to generate a blend between things. An image of a dog/elephant hybrid, then an image of a dog/elephant hybrid, etc.

**Reply by mashb1t** [reply]
Alright, then you're looking for the attention shift of A1111, second link!

**Comment by bblackmoor** [comment]
See also: https://github.com/lllyasviel/Fooocus/discussions/1380#discussioncomment-8764374
